core/src/test/java/com/sequenceiq/cloudbreak/service/freeipa/FreeipaServiceTest.java

```diff
@@ -0,0 +1,70 @@
+package com.sequenceiq.cloudbreak.service.freeipa;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.sequenceiq.cloudbreak.common.exception.CloudbreakServiceException;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.common.AvailabilityStatus;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.common.Status;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.describe.DescribeFreeIpaResponse;
+
+@ExtendWith(MockitoExtension.class)
+class FreeipaServiceTest {
+
+    private static final String ENV_CRN =
+            "crn:cdp:environments:us-west-1:460c0d8f-ae8e-4dce-9cd7-2351762eb9ac:environment:6b2b1600-8ac6-4c26-aa34-dab36f4bd243";
+
+    @Mock
+    private FreeipaClientService freeipaClientService;
+
+    @InjectMocks
+    private FreeipaService underTest;
+
+    @Test
+    void testCheckFreeipaRunningWhenFreeIpaStoppedThenReturnsFalse() {
```

Comment:
nit: here one could test all not available Statuses that freeipa availability checker returns false for all of them.